Automattic/newspack-blocks
# fix: guest author selection in Author Profile block

### All Submissions:






### Changes proposed in this Pull Request:



Fixes a bug that prevents selection of guest authors via autocomplete suggestion in the Author Profile block.

Also bumps `newspack-components` to v2.1.0, which includes [this fix](https://github.com/Automattic/newspack-plugin/pull/1679).

Closes #1184.

### How to test the changes in this Pull Request:

Confirm that #1184 is not reproducible and that you can search for and select both guest authors and WP users via the autocomplete field.

### Other information:
